flepimop/gempyor_pkg/tests/seir/test_model_info.py

```diff
@@ -44,7 +44,10 @@ def test_ModelInfo_init_tf_is_ahead_of_ti_fail(self):
         config.read(user=False)
         config.set_file(f"{DATA_DIR}/config_test.yml")
         config["start_date"] = "2022-01-02"
-        with pytest.raises(ValueError, match=r"tf.*is less than or equal to ti.*"):
+        with pytest.raises(
```

Comment:
I'd prefer this match to the error-causing values (to verify that the error contains the offending values for the user to see), rather than the precise wording of the message.

Reply:
Note to self: need to find how `ti` and `tf` are referenced in this file